docs/Dev_env_setup.rst

```diff
@@ -0,0 +1,116 @@
+Development environment setup
+===============================
+
+Here are some guidelines on how to set up your development environment for the 
+lago project.
+
+
+Requirements
+-------------
+
+You'll need some extra packages to get started with the code for lago, assuming
+you are runnig fedora::
+
+  > sudo dnf install git mock libvirt-daemon qemu-kvm autotools
+
+And you'll need also a few python libs, you can install them from the repos or
```

Comment:
python -> Python
'you can install them from' -> 'which you you can install from'